extensions/amp-user-notification/0.1/amp-user-notification.css

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-user-notification {
+  visibility: hidden;
+}
+
+.amp-user-notification {
+  position: fixed;
+  top: 0;
+  left: 0;
+  background: inherit;
+  overflow: hidden;
+  background: rgba(255, 255, 255, .60);
+  margin-bottom: 8px;
+  z-index: 9998;
```

Comment:
adjusted to `z-index: 1`, the notification gets clipped off by `amp-img`/`amp-iframe` because they have `absolute` positioned children.